homeworks/serhiy_shchur/homework_1/src/components/CommentsList.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentsList.css';
+
+export default class CommentsList extends React.Component {
+    state = {
+        comments: [],
+        author: "",
+        commentBody: ""
+    };
+
+    handleAuthorChange(e) {
+        this.setState({
+            author: e.target.value
+        });
+    }
+
+    handleCommentBodyChange(e) {
+        this.setState({
+            commentBody: e.target.value
+        });
+    }
+
+    handleCommentDelete(e, delIndex) {
+        e.preventDefault();
+        this.setState({comments: this.state.comments.filter((_, i) => i !== delIndex)});
+    }
+
+    addComment(e) {
+        e.preventDefault();
+        if(this.state.author && this.state.commentBody) {
+            this.setState({
+                comments: [...this.state.comments, {author: this.state.author, commentBody: this.state.commentBody}],
```

Comment:
better to provide comment object from child component

Reply:
@lavriv92 not sure what do you mean by that

Reply:
I've made child component as functional component so it just can take props from parent and render itself